src/comm/LinkManager.cc

```diff
@@ -479,6 +479,28 @@ void LinkManager::_updateAutoConnectLinks(void)
         createConnectedLink(config);
         emit linkConfigurationsChanged();
     }
+#ifndef __mobile__
+    // check to see if nmea gps is configured for UDP input, if so, set it up to connect
+    if (_autoConnectSettings->autoConnectNmeaPort()->cookedValueString() == "UDP Port")
+    {
```

Comment:
Minor nit, but coding style should have this on the previous line.

Reply:
sorry, fixed.